virttest/libvirt_xml/vm_xml.py

```diff
@@ -3223,7 +3223,8 @@ class VMFeaturesXML(base.LibvirtXMLBase):
                  'hyperv_reenlightenment_state', 'hyperv_vpindex_state',
                  'kvm_hidden_state', 'pvspinlock_state', 'smm', 'hpt',
                  'htm', 'smm_tseg_unit', 'smm_tseg', 'nested_hv',
-                 'pmu', 'kvm_poll_control', 'ioapic')
+                 'pmu', 'kvm_poll_control', 'ioapic',
+                 'kvm_dirty_ring_state', 'kvm_dirty_ring_size')
```

Comment:
please update doc string after line 3218 to match attributes change

Reply:
Updated. Thanks